src/components/EventForm/EventForm.js

```diff
@@ -0,0 +1,154 @@
+      width: 600,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+  },
+  paper: {
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(3),
+    padding: theme.spacing(2),
+    [theme.breakpoints.up(600 + theme.spacing(3) * 2)]: {
+      marginTop: theme.spacing(6),
+      marginBottom: theme.spacing(6),
+      padding: theme.spacing(3),
+    },
+  },
+  stepper: {
+    padding: theme.spacing(3, 0, 5),
+  },
+  buttons: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  button: {
+    marginTop: theme.spacing(3),
+    marginLeft: theme.spacing(1),
+  },
+}));
+
+
+
+
+export default function MyForm() {
+  const classes = useStyles();
+
+
+  const validationSchema = Yup.object({
+    email: Yup.string().email().required(),
+    name: Yup.string().required(),
+    id: Yup.number('Valid Student ID required')
+        .min(9999999, 'Valid Student ID required')
+        .max(100000000, 'Valid Student ID required')
+        .required(),
+    password: Yup.string().min(6).required(),
+    confirmPassword: Yup.string()
+        .oneOf([Yup.ref('password'), null], "Password must match")
+        .required("Please confirm your password"),
+    firstname: Yup.string().required("First name is required"),
+    lastname: Yup.string().required("Last name is required"),
+  });
+
+  const initialValues = { email: "", name: "", id: "", password: "", confirmPassword: "", firstname: "", lastname: "" };
+
+
+  return (
+    <React.Fragment>
+      {/* <CssBaseline />
+      <AppBar position="absolute" color="default" className={classes.appBar}>
+        <Toolbar>
+          <Typography variant="h6" color="inherit" noWrap>
+            Company name
+          </Typography>
+        </Toolbar>
+      </AppBar> */}
+
+    
+      <main className={classes.layout}>
+        <Paper className={classes.paper}>
+          <Typography component="h1" variant="h4" align="center">
+            Coding for Noobs 1 with Lighthouse Labs
+          </Typography>
+          <br></br>
+
+          {/* <FormBody/> */}
+          <Formik
+            render={props => <Form {...props} />}
+            initialValues={initialValues}
+            validationSchema={validationSchema}
+            onSubmit={submitValues}
+          />
+          <Button
+            variant="contained"
+            color="primary"
+            onClick="TODO"
+            className={classes.button}
+          >
+            Submit
+          </Button>
+
+        </Paper>
+        <Copyright />
+      </main>
+    </React.Fragment>
+  );
+  async function submitValues(values) {
+      const { email, name, id, password } = values;
+  
+      try {
+          // await Auth.MyForm({
+          //     username: email,
+          //     password,
+          //     attributes: {
+          //         email,
+          //         name,
+          //         nickname: id
+          //     },
+          // })
+          console.log(values);
```

Comment:
This doesn't need to be wrapped in a try catch

Reply:
Just to confirm: is "await Auth" for database validation? So i can safely remove that as well?

Reply:
Auth is for logging in to AWS

Reply:
Yeah remove it (and the commented out import)